run.py

```diff
@@ -483,6 +484,7 @@ def fetch_test_details(var) -> dict:
     tests = suite.get('tests')
     tcs = []
     jenkins_rc = 0
+    fail_count = 0
```

Comment:
Hope this may not be needed, since we already receiving `PASSED` or `FAILED` in the email header. 
But I think we may need another one `SETUP_ERROR` which indicates environment issue. what do you think?

Reply:
so you mean `SETUP_ERROR` in the mail header. that's sound right. but it would also be nice have the failed or passed count in the summary.

Reply:
Yes, or like `ABORTED` or `SETUP_FAILURE`.

Reply:
Added "SETUP_FAILURE" in the email header if there is a error in environment setup.